scripts/start.js

```diff
@@ -33,7 +34,8 @@ compiler.plugin('done', function (stats) {
 
     console.log(chalk.green('Compiled successfully!'));
     console.log('\nThe app is running at:');
-    console.log('\n    ' + chalk.cyan('http://localhost:' + port + '/'));
+    // TODO: display the correct port, if the default is busy.
+    console.log('\n    ' + chalk.cyan('http://localhost:' + DEFAULT_PORT + '/'));
```

Comment:
@fobos I like the implementation, but I also have a couple concerns.

According to the original idea:

- Specify port from a flag `-p` or `--port` #36 
- Prompt user about occupied port, and ask for `Y/n` answer  
(this is how it's done in create-react-app and I think it makes sense)
- If port is occupied and user answered `Y` to the prompt, auto-detect and use any available free port

On top of that, all the code from `start.js` will be available to the user, so it must be as simple as it is possible.

I think it could be simplified, if we'll keep a variable with port in the module scope, and update it from the callback:

```js
var usedPort;

detect(DEFAULT_PORT, function(err, unoccupiedPort) {

  // That way we can reduce the use of promises
  usedPort = unoccupiedPort;
})
```

I personally don't have any problems with promises, but I suspect, that some people might have.

Please tell me what you think about that and thanks a lot for your effort! 🙌